common/src/main/scala/net/psforever/objects/Vehicle.scala

```diff
@@ -145,7 +145,9 @@ class Vehicle(private val vehicleDef : VehicleDefinition) extends AmenityOwner
   }
 
   def Health_=(assignHealth : Int) : Int = {
-    health = math.min(math.max(0, assignHealth), MaxHealth)
+    if(health != 0) {
```

Comment:
Lets make an `isDead` bool instead (might need to check it elsewhere in the object).

Reply:
Done